packages/design-system-react-native/src/components/AvatarGroup/AvatarGroup.stories.tsx

```diff
@@ -0,0 +1,158 @@
+import { ScrollView, View } from 'react-native';
+import type { Meta, StoryObj } from '@storybook/react-native';
+
+import { AvatarGroupSize } from '../../shared/enums';
+import AvatarGroup from './AvatarGroup';
+import {
+  DEFAULT_AVATARGROUP_PROPS,
+  SAMPLE_AVATARGROUP_AVATARACCOUNTPROPSARR,
+  SAMPLE_AVATARGROUP_AVATARFAVICONPROPSARR,
+  SAMPLE_AVATARGROUP_AVATARNETWORKPROPSARR,
+  SAMPLE_AVATARGROUP_AVATARTOKENPROPSARR,
+} from './AvatarGroup.constants';
+import type { AvatarGroupProps } from './AvatarGroup.types';
+import { AvatarGroupVariant } from './AvatarGroup.types';
+
+const meta: Meta<AvatarGroupProps> = {
+  title: 'Components/AvatarGroup',
+  component: AvatarGroup,
+  argTypes: {
+    variant: {
+      control: 'select',
+      options: AvatarGroupVariant,
+    },
+    size: {
+      control: 'select',
+      options: AvatarGroupSize,
+    },
+    max: {
+      control: 'number',
+    },
+    isReverse: {
+      control: 'boolean',
+    },
+    twClassName: {
+      control: 'text',
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<AvatarGroupProps>;
+
+const AvatarGroupStory: React.FC<Omit<AvatarGroupProps, 'avatarPropsArr'>> = ({
+  variant,
+  ...props
+}) => {
+  switch (variant) {
+    case AvatarGroupVariant.Account:
+      return (
+        <AvatarGroup
+          {...props}
+          variant={variant}
+          avatarPropsArr={SAMPLE_AVATARGROUP_AVATARACCOUNTPROPSARR}
+        />
+      );
+    case AvatarGroupVariant.Favicon:
+      return (
+        <AvatarGroup
+          {...props}
+          variant={variant}
+          avatarPropsArr={SAMPLE_AVATARGROUP_AVATARFAVICONPROPSARR}
+        />
+      );
+    case AvatarGroupVariant.Network:
+      return (
+        <AvatarGroup
+          {...props}
+          variant={variant}
+          avatarPropsArr={SAMPLE_AVATARGROUP_AVATARNETWORKPROPSARR}
+        />
+      );
+    case AvatarGroupVariant.Token:
+      return (
+        <AvatarGroup
+          {...props}
+          variant={variant}
+          avatarPropsArr={SAMPLE_AVATARGROUP_AVATARTOKENPROPSARR}
+        />
+      );
+  }
+};
+
+export const Default: Story = {
+  args: {
+    variant: AvatarGroupVariant.Favicon,
+    size: DEFAULT_AVATARGROUP_PROPS.size,
+    max: DEFAULT_AVATARGROUP_PROPS.max,
+    isReverse: DEFAULT_AVATARGROUP_PROPS.isReverse,
+    twClassName: '',
+  },
+  render: (args) => <AvatarGroupStory {...args} />,
+};
```

Comment:
non-blocking: Not sure if this is intended or needs to be fixed but when `isReverse` is true it aligns differently to the page 

<img width="300" alt="Screenshot 2025-03-07 at 2 11 21 PM" src="https://github.com/user-attachments/assets/9777ba1b-842c-45a3-b91f-6fae2caf513d" /><img width="300" alt="Screenshot 2025-03-07 at 2 11 15 PM" src="https://github.com/user-attachments/assets/2affde81-eee7-48d8-8955-6f1a0ab585ec" />

Reply:
This is the result of row vs row-reverse so it's working as intended